datamade/erikson-edi
# Support custom cutoffs for Overlays

## Overview

This PR adds a new attribute, `Overlay.cutoffs`, allowing the user to specify custom cutoffs (or bins) for a given overlay in the map UI.

## Testing instructions

* Run `./manage.py migrate` to update models
* Navigate to http://127.0.0.1:8000/admin/edi/overlay/
* Using the cutoffs sheet that Veronica supplied for us, go through the overlays and fill out the `Cutoffs` field for each overlay by copy/pasting the raw CSV values
    * When I did this, I noticed that the `import_geographies` management command was erroneously assigning a scale of `count` instead of `percentage` for `Homicide rate` in GESTL; if this is true for you as well, make sure to fix it when you update the cutoffs or else the display will be wonky
* Navigate to the frontend maps, test all the overlays, and confirm that the circles on the map and the legend both look good
